renovate.json

```diff
@@ -27,112 +19,62 @@
     "prCreation": "immediate",
     "prPriority": 5
   },
-  "major": {
-    "labels": [
-        "Dependencies",
-        "Renovate"
-    ],
-    "prCreation": "not-pending",
-    "prPriority": 0
-  },
-  "prHourlyLimit": 1,
+  "prHourlyLimit": 10,
```

Comment:
Would it be preferable for this to be lower lest we be flooded by renovate PRs again?

Reply:
I have lowered it.